src/frontend/src/hooks/finance.hooks.ts

```diff
@@ -61,6 +62,10 @@ export interface ExpenseTypePayload {
   allowedRefundSources: ClubAccount[];
 }
 
+export interface EditVendorTypePayload {
```

Comment:
just EditVendorPayload, ExpenseType was an alternate name for an Account Code that accidentally got thrown in the mix, I've been trying to change it to AccountCode wherever I can, but type is a part of the finance term, not smthn we added in this case